### PR TITLE
Separated Configuration Cucumber and Running Cucumber

### PR DESCRIPTION
**Pull request by tsundberg**
Configuration and execution are two different topics. They where combined. I separated them into two pages. I also updated links to point to the new page instead of the old configuration page.

### CODE REVIEW

File: content/about/contributing.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ If you are certain you have found a bug, just register a ticket with the relevan
 ## Help us help you
 
 You will likely use Cucumber combined with various web frameworks, many different dependencies and your own code.
-You can [run Features](/configuration/#running-features) in different ways, and many things can go wrong - and quite often it has nothing to do with Cucumber at all.
+You can [run Features](/running/) in different ways, and many things can go wrong - and quite often it has nothing to do with Cucumber at all.
```
Comment (mlvandijk) [inline content/about/contributing.md]: Change "run Features" -> "run features" (for consistency) or "run Cucumber" like in the menu

File: content/cucumber/environment-variables.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ title: Environment Variables
 
 > TODO: Make more general
 
-When [Running Features](/configuration/#running-features), it can sometimes be handy to pass special
+When [Running Features](/running), it can sometimes be handy to pass special
```
Comment (mlvandijk) [inline content/cucumber/environment-variables.md]: Change "Running Features" to "running features" (lowercase for consistency) or "running Cucumber" (like menu)
"Step Definitions" -> "step definitions" (same)

Comment (mlvandijk) [reply content/cucumber/environment-variables.md]: (Probably all of the Ruby docs should be checked for consistency)

File: content/implementations/ruby/ruby-on-rails.md [hunk 3]
```diff
@@ -163,7 +163,8 @@ default: --format pretty
 html: --format html --out features.html
 ```
 
-Remember that you need `AUTOFEATURE=true` for autospec to include Cucumber Features. See [Running Features](/configuration/#running-features) and [Autotest Integration](/wiki/autotest-integration) for more info.
+Remember that you need `AUTOFEATURE=true` for autospec to include Cucumber Features. See [Running Features](/running/)
```
Comment (mlvandijk) [inline content/implementations/ruby/ruby-on-rails.md]: lowercase for "Cucumber Features" to "Cucumber features".
lowercase for "Autotest Integration" to "autotest integration" (unless Autotest is a product name? then it should be "Autotest integration"?)

File: content/installation.md [hunk 4]
```diff
@@ -46,9 +46,9 @@ repositories {
 }
 ```
 
-You can now use cucumber's [CLI Runner](/configuration/#configuration) to execute by adding a `cucumber` task to `build.gradle`.
+You can now use cucumber's [CLI Runner](/running/#gradle-runner) to execute by adding a `cucumber` task to `build.gradle`.
 
-For more information, see [Configuration](/configuration).
+For more information, see [Running](/running).
```
Comment (mlvandijk) [inline content/installation.md]: Change to "[Running Cucumber](/running)." (like menu)?

File: content/tags.md [hunk 5]
```diff
@@ -123,7 +123,7 @@ cucumber --tags "@smoke and @fast"
 <p></p>
 
 {{% tip "Filtering by line" %}}
-Another way to run a subset of scenarios is to use the `file.feature:line` pattern or the `--scenario` option as described in [Running Features](/configuration/#running-features).
+Another way to run a subset of scenarios is to use the `file.feature:line` pattern or the `--scenario` option as described in [Running Features](/running).
```
Comment (mlvandijk) [inline content/tags.md]: Change link "[Running Features]" to "Running Cucumber"?

File: content/wiki/autotest-integration.md [hunk 6]
```diff
@@ -53,9 +53,11 @@ autotest: --color
 autotest-all: --color
 ```
 
-Please see [Running Features](/configuration/#running-features) for more information about profiles, and the [Ruby on Rails](/implementations/ruby/ruby-on-rails/) page for extra help when running in that context.
+Please see [Running Features](/running/) for more information about profiles, and the
```
Comment (mlvandijk) [inline content/wiki/autotest-integration.md:56]: Change link "[Running Features]" to "Running Cucumber"?

File: content/wiki/autotest-integration.md [hunk 7]
```diff
@@ -109,7 +111,7 @@ It's useful to use the Pretty formatter in autotest to see exactly what's going
 
 If you define an `autotest-all` profile, autotest will use this profile instead when running your Features. You may want to use the Progress formatter for `autotest-all`.
 
-For more information on defining profiles, see [Running Features](/configuration/#running-features).
+For more information on defining profiles, see [Running Features](/running/).
```
Comment (mlvandijk) [inline content/wiki/autotest-integration.md]: Change link "[Running Features]" to "Running Cucumber"?

**Review by mlvandijk — CHANGES_REQUESTED** [review]
Thanks @tsundberg for making this improvement! Minor changes requested for consistency only: using lowercase and a consistent title for the new page.

File: content/about/contributing.md (hunk 1 above)
Comment (tsundberg) [reply content/about/contributing.md]: I'm pretty sure I didn't touch "run Features". But I agree and fixing your suggestions is a good thing to do.

File: content/implementations/ruby/ruby-on-rails.md (hunk 3 above)
Comment (tsundberg) [reply content/implementations/ruby/ruby-on-rails.md]: Unsure bout Autotest or autotest. I'll lowercase it for now.

File: content/wiki/autotest-integration.md (hunk 7 above)
Comment (tsundberg) [reply content/wiki/autotest-integration.md]: Done

File: content/wiki/autotest-integration.md (hunk 6 above)
Comment (tsundberg) [reply content/wiki/autotest-integration.md:56]: Done

File: content/tags.md (hunk 5 above)
Comment (tsundberg) [reply content/tags.md]: Done

File: content/installation.md (hunk 4 above)
Comment (tsundberg) [reply content/installation.md]: Done

File: content/cucumber/environment-variables.md (hunk 2 above)
Comment (tsundberg) [reply content/cucumber/environment-variables.md]: Done

File: content/about/contributing.md (hunk 1 above)
Comment (mlvandijk) [reply content/about/contributing.md]: Yes, I know. Thanks for fixing anyway :)

**Review by mlvandijk — APPROVED** [review]

**Pull request merged**